lib/operations/command_v2.js

```diff
@@ -15,10 +15,13 @@ class CommandOperationV2 extends OperationBase {
   constructor(parent, options, operationOptions) {
     super(options);
 
+    const noInherit = this.hasAspect(Aspect.NO_INHERIT_OPTIONS);
+    const inherit = noInherit ? {} : parent;
```

Comment:
Nit: this could just be one line, `const inherit = this.hasAspect(Aspect.NO_INHERIT_OPTIONS) ? {} : parent;`, I'm generally not a huge fan of variable declarations that are only used once like this.